backend/db/utils.py

```diff
@@ -703,19 +703,23 @@ def reset_temp_refresh_tables(schema: str = SCHEMA):
           file=sys.stderr)
     with get_db_connection(schema=schema) as con:
         tables: List[str] = list_tables(con, schema)
```

Comment:
## `list_tables()` vs `show_tables()`
2 funcs that do roughly same thing. My bad, I did `list_tables()` this 6 months after Siggie did `show_tables()`. Some differences:

`list_tables()`
- Has `filter_temp_refresh_tables` (default: `False`) param to filter tables ending w/ `_old` or `_new`
- By default, returns simple list of table names
- No option to show views
- Can select `schema`

`show_tables()`
- Shows more info, i.e. schema, table name, whether table or view, creator (e.g. 'n3c', 'all_csets', 'table', 'thadmin').
- Prints info by default
- Only `n3c` schema

I guess I should refactor to use `show_tables()` instead, and integrate some of my functionality into it.